scripts/hid_configurator/README.rst

```diff
@@ -17,23 +18,19 @@ It can be used for the following purposes:
 Overview
 ********
 
-The script looks for a nRF Desktop device whose name is passed as argument.
-The device must be connected to the host through USB or Bluetooth and it must match the Product ID (PID) and Vendor ID (VID) associated with the device name.
-If the device cannot be found, the script looks for a nRF Desktop dongle and tries to connect to the device through the dongle.
+The script looks for nRF Desktop devices connected directly to the host through USB/Bluetooth or via nRF Desktop dongle.
```

Comment:
```suggestion
The script looks for nRF Desktop devices connected directly to the host through USB, Bluetooth, or nRF Desktop dongle.
```

Reply:
I will remove word `directly` here - device is connected directly through the USB, Bluetooth or through the dongle.